Autopilot/AttitudeManager/OrientationControl.h

```diff
@@ -1,233 +1,107 @@
- *
- * Output:  None.
- *
- *****************************************************************************/
-
-void setIntegralSum(unsigned char YPR, float value);
-
-/*****************************************************************************
- * Function: float getIntegralSum(unsigned char YPR)
- *
- * Preconditions: None.
- *
- * Overview: This function is used get the value of the integrator (from the
- * PID loops). This is often used to check the integrator for testing and
- * automatic checking/limiting.
- *
- * Input:   unsigned char YPR -> The value indicating the pitch, roll, or yaw
- *              components. See main.h for their definitions.
- *
- * Output:  float -> The value of the requested integral sum.
- *
- *****************************************************************************/
-
-float getIntegralSum(unsigned char YPR);
-
-/*****************************************************************************
- * Function: float getGain(unsigned char YPR, unsigned char type)
- *
- * Preconditions: None.
- *
- * Overview: This function is used get the value of the gains (from the
- * PID loops). This is often used to check the gains for testing and
- * automatic checking/limiting.
- *
- * Input:   unsigned char YPR -> The value indicating the pitch, roll, or yaw
- *              components. See main.h for their definitions.
- *          unsigned char type -> The value indicating the type of gain (KP,KI,KD)
- *              to return. See the definitions at the top of this header file.
- *
- * Output:  float -> The value of the requested gain.
- *
- *****************************************************************************/
-
-float getGain(unsigned char YPR, unsigned char type);
-
-/*****************************************************************************
- * Function: void setGain(unsigned char YPR, unsigned char type, float value)
- *
- * Preconditions: None.
- *
- * Overview: This function is used set the value of the gains (for the
- * PID loops). This is often used to alter the gains for testing and
- * automatic checking/limiting.
- *
- * Input:   unsigned char YPR -> The value indicating the pitch, roll, or yaw
- *              components. See main.h for their definitions.
- *          unsigned char type -> The value indicating the type of gain (KP,KI,KD)
- *              to return. See the definitions at the top of this header file.
- *          float -> The value of the selected gain.
- *
- * Output:  None.
- *
- *****************************************************************************/
-
-void setGain(unsigned char YPR, unsigned char type, float value);
-char areGainsUpdated();
+
+#include "main.h"
+
+#ifndef ORIENTATIONCONTROL_H
+#define	ORIENTATIONCONTROL_H
+
+#define PID_CHANNELS 8 // to start
+
+#define PID_RESET_TIME 1000 // timeout to reset I and D terms (ms)
+
+#define MAX_ROLL_ANGLE 35.f // degrees
+#define MAX_PITCH_ANGLE 35.f
+
+#define MAX_ROLL_RATE 180.f // degrees/second
```

Comment:
Not really. There were max roll & pitch angles before, these are based off those + other flight controllers.